.cargo/config.toml

```
@@ -4,6 +4,8 @@ ci-clippy = "clippy --all-targets --all-features -- -D warnings"
 ci-license = "license-template --template .license_template"
 ci-udeps = "udeps --all-targets --backend=depinfo"
 ci-udeps-external = "udeps --all-targets --manifest-path external-crates/move/Cargo.toml --backend=depinfo"
+ci-deny = "deny check --hide-inclusion-graph"
+ci-deny-external = "deny --manifest-path external-crates/move/Cargo.toml check --hide-inclusion-graph"
```

```suggestion
ci-deny-external = "deny check --manifest-path external-crates/move/Cargo.toml --hide-inclusion-graph"
```
More readable in that order IMO

---

Yes, but this is not how that command works. `deny` accepts the manifest path and `check` is a subcommand which does not.

```
➜  iota git:(dev-tools/external-deny) ✗ cargo deny check --manifest-path external-crates/move/Cargo.toml --hide-inclusion-graph
error: unexpected argument '--manifest-path' found
```

---

Ah alright, my bad. Ewww tho